code/game/objects/items/defib.dm

```diff
@@ -517,7 +526,7 @@
 			T.audible_message("<span class='warning'>\The [defib] lets out an urgent beep and lets out a steadily rising hum...</span>")
 		else
 			user.audible_message("<span class='warning'>[src] let out an urgent beep.</span>")
-		if(do_after(user, 30, target = H)) //Takes longer due to overcharging
+		if(do_after(user, 15, target = H)) //Takes longer due to overcharging
```

Comment:
doc?

Reply:
oh yeah i implemented what PK wanted but forgot to document it. whoops

Reply:
> doc?

docced